File: Codeforces/1676B-Equal_Candies.cpp
#include <bits/stdc++.h>
using namespace std;

int solve(vector<int>& A, int n) {
  int total = 0;

  auto it = min_element(A.begin(), A.end());
  int min = *it;  

  for (int val : A) {
    if (val != min) total += val - min;
  }
  return total;
}

int main() {
  int t;
  cin >> t;

  while (t--) {
    int n;
    cin >> n;

    vector<int> A(n);
    for (int& x : A) cin >> x;
    
    int ans = solve(A, n);
    cout << ans << "\n";
  }
}

File: Codeforces/1676B-Equal_Candies.py
def solve(arr, n):
    min_val = min(arr)
    total = 0

    for val in arr:
        if val != min_val:
            total += val - min_val
    return total

for _ in range(int(input())):
    n = int(input())
    arr = list(map(int, input().split()))

    print(solve(arr, n))
